dynamiqs/mesolve/mesolve.py

```diff
@@ -182,11 +182,11 @@ def _check_mesolve_args(
     H: TimeArray, jump_ops: list[TimeArray], rho0: Array, exp_ops: Array | None
 ):
     # === check H shape
-    check_shape(H, 'H', '(..., n, n)')
+    check_shape(H, 'H', '(..., n, n)', subs={'...': '...H'})
```

Comment:
"H must be of shape `...H`" sounds a bit cryptic, even though I see what you mean. I would replace with `(..., n, n)`. (PS: I don't have the "suggesion option anymore in GH interface)

Reply:
I quite like `...H` on my end! I think it's quite explicit in the sense that I can't find another interpretation for it, so people should understand.

Reply:
Could be useful to add a short sentence about what `...H` and `...psi0` mean in the "Running multiple simulations concurrently" box above?

Reply:
@abocquet the `subs` argument only replaces the dictionnary keys substring, not the full shape specification. The resulting error message would read
```python
ValueError: Argument `H` must have shape (...H, n, n), but has shape H.shape=(2,).
```

You can test with e.g.
```python
from dynamiqs._checks import check_shape
import jax.numpy as jnp
H = jnp.array([1, 2])
check_shape(H, 'H', '(..., n, n)', subs={'...': '...H'})
```

Reply:
> Could be useful to add a short sentence about what ...H and ...psi0 mean in the "Running multiple simulations concurrently" box above?

I'm assuming that people know what we mean by this notation (we use it everywhere e.g. in the utils). I think if we want to document it, we should ideally do it in a single place, not in every functions. But IMO it's pretty self-explanatory.